README.md

```diff
@@ -140,6 +141,30 @@ client = Client.new(url, schema_path: 'config/your_graphql_schema.json')
 client.schema.dump! # you only need to call this when graphql schema changes
 ```
 
+### Preloading schema once
+
+Even if caching the schema on disk, instanciating `Graphlient::Client` often can be both time and memory intensive due to reading in the schema for each instance. This is especially true if the schema is a large file. To get around these performance issues, instanciate your schema once and pass it in as a configuration option.
```

Comment:
I think it's `instantiating` and `instantiate` with a t. Amiright?

memory intensive -> memory consuming
reading in -> loading

Reply:
😳 Great catch!